metadeploy/api/models.py

```diff
@@ -947,10 +947,14 @@ def get_from_session(self, session):
         """
         uuid = session.get("scratch_org_id")
         try:
-            return self.get(uuid=uuid, status=self.model.Status.complete)
+            return self.get(uuid=uuid)
```

Comment:
I'm not sure if this will change the behavior of others places where we call `get_from_session`, as they might expect completed orgs only. This is just a hunch from when I did this refactor, so hopefully the the tests will catch anything.

Reply:
I do prefer it this way btw, `get_from_session` makes much more sense when it only depends on the session. Other places can refine the query by chaining `filter` as I described in another comment

Reply:
Good catch; I found a few places where I think we do want to limit to completed orgs, and made those changes.